bin/generate-starter-schema.py

```diff
@@ -81,7 +81,12 @@ def convert_row_to_property(row, sheet_has_autopopulate):# noqa
     rule_ref = spec.get(Columns.RULE_REFERENCE.name)
 
     token = title.replace("\n", "_").replace(" ", "_").replace(".", "")\
-        .replace("(", "").replace(")", "")
+        .replace("(", "").replace(")", "").replace("/", "_")\
+        .replace("__", "_").lower()
+    # Prepend tokens that start with a number (presumed to be a line number)
+    # with capital letter "L".
+    if token[0].isdigit():
+        token = 'L' + token
```

Comment:
since we're keeping the tokens lowercase, do we want to make this a lowercase `l`?

Reply:
No, I did the lowercase "l" at first but it was too confusing next to the number "1" or "l" in the original property name. Uppercase "L" is more readable so let's make an exception in this case